src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+// Action types
+const ADD = 'bookstore/books/ADD';
+const REMOVE = 'bookstore/books/REMOVE';
+
+// Action creators
+export const add = (book) => ({
+  type: ADD,
+  payload: book,
+});
+
+export const remove = (index) => ({
+  type: REMOVE,
+  payload: index,
+});
+
+// Reducer
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD:
+      return [...state, action.payload];
+    case REMOVE:
```

Comment:
- [ ] nice working on your `remove book functionality`, Kindly check when I click the `delete button` it adds a book instead of removing the book.